tests/test_ilastik/test_applets/edgeTraining/testEdgeTrainingSerializer.py

```diff
@@ -0,0 +1,134 @@
+import h5py
+import numpy
+import pandas
+import pytest
+
+from ilastikrag import Rag
+from ilastik.applets.edgeTraining import OpEdgeTraining
+from lazyflow.utility.testing import build_multi_output_mock_op, SlotDescription
+from ilastik.applets.edgeTraining.edgeTrainingSerializer import EdgeTrainingSerializer
+import vigra
+
+
+@pytest.fixture
+def empty_project_file():
+    with h5py.File("test_project1", mode="a", driver="core", backing_store=False) as test_project:
+        test_project.create_dataset("ilastikVersion", data=b"1.0.0")
+        yield test_project
+
+
+@pytest.fixture
+def superpixels():
+    # superpixels needed for creation and deserialization of the rag
+    data = numpy.zeros((5, 6, 1), dtype="uint32")
+    data[0:2, ...] = 1
+    data[2:, ...] = 2
+    return data
+
+
+@pytest.fixture
+def rag(superpixels):
+    return Rag(vigra.taggedView(superpixels, axistags="yxc").withAxes("yx"))
+
+
+@pytest.fixture
+def inputs(graph, superpixels):
+    slot_data = {
+        "WatershedSelectedInput": SlotDescription(
+            level=1, dtype=numpy.float32, shape=(5, 6, 1), axistags=vigra.defaultAxistags("yxc")
```

Comment:
I wonder if it's better to accept `str` instead of `vigra.axistags` in `SlotDescription`, to avoid repeating `vigra.defaultAxistags` in test code.

Reply:
makes it more convenient to use, for sure